scormxblock/scormxblock.py

```diff
@@ -133,6 +135,9 @@ def s3_upload(all_content, temp_directory, dest_dir):
             ExtraArgs={'ACL': 'public-read', 'ContentType': content_type},
         )
 
+    if not isinstance(temp_directory, str):
+        temp_directory.close()
```

Comment:
This operation is not within the function's scope, why did you move it here?. @alexjmpb